.gitattributes

```diff
@@ -1 +1 @@
-tests/samples/* linguist-vendored
+tests/samples/**/*.html linguist-vendored
```

Comment:
Shouldn’t it be just `tests/samples/**`?

Reply:
Ah, hadn't seen there were other files too (though they don't get included in linguist by default). I will change it, makes more sense.